Docs/Book/Cartridge.md

```diff
@@ -171,7 +171,7 @@ Given we're searching through 1.7 million compounds these search times aren't in
 
 One easy way to speed things up, particularly for queries that return a large number of results, is to only retrieve a limited number of results:
 
-    chembl_23=# select * from rdk.mols where m@>'c1cccc2c1CNCCN2' limit 100;
+    chembl_25=# select * from rdk.mols where m@>'c1cccc2c1CNCCN2' limit 100;
      molregno |                                                                                             m                                                             
 
     ----------+-----------------------------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
Don't the results returned from this query (and the next one) change as well?

Reply:
updated